eclass/perl-module.eclass

```diff
@@ -319,7 +276,7 @@ perl-module_src_compile() {
 	perl_set_version
 
 	case ${EAPI} in
-		6|7)
+		7)
 			if [[ $(declare -p mymake 2>&-) != "declare -a mymake="* ]]; then
 				local mymake_local=(${mymake})
 			else
```

Comment:
below: emake || die (also, let's just use edo instead of the einfo there)

Reply:
`|| die` removed, but no `edo` since `emake` echos its command anyway